cli/cli/commands/lsp/resource/kurtosis_starlark.json

```diff
@@ -554,6 +554,130 @@
           "type": "ReadyCondition",
           "content": "ready_conditions?",
           "detail": "This field can be used to check the service's readiness after this is started to confirm that it is ready to receive connections and traffic"
+        },
```

Comment:
nice! lsp hasn't been updated in a while, should make a ticket to support missing configs